src/controllers/VariableController.ts

```diff
@@ -235,10 +233,10 @@ export class VariableController {
      * @param id the id of the variable to update
      * @param source the new variable source
      */
-    setSource = async (id: string, source?: ImageVariableSource) => {
+    setSource = async (id: string, source?: ConnectorImageVariableSource) => {
```

Comment:
```suggestion
```

`setSource` is marked as deprecated. The only reason it is still here is because `UrlImageVariableSource` was still there. Now that it will be removed from the engine, I believe this method can be completely removed. 

We already had at least one SDK release where the deprecation was added so should be fine to remove it? 

Thoughts? @Dvergar @brapoprod

Reply:
I had this thought about maybe giving integrators the time to adapt.

But If they need to refactor their code anyway (new image var source type) I would vote to remove it altogether.

Reply:
_Resolved as Brian approved the removal via chat._